core/server/common/src/main/java/alluxio/master/journal/ufs/UfsJournal.java

```diff
@@ -220,6 +224,7 @@ synchronized void write(JournalEntry entry) throws IOException, JournalClosedExc
    */
   @VisibleForTesting
   public synchronized void flush() throws IOException, JournalClosedException {
```

Comment:
is this one not used....?

Reply:
it's only used in testing to make sure journals have been flushed before the test program does assertions/ checks conditions.

Reply:
interesting... i added some comments on this